kernel/src/main/java/org/kframework/backend/kore/ModuleToKORE.java

```diff
@@ -160,6 +162,9 @@ public String convert(boolean heatCoolEq) {
             }
         }
         for (Production prod : iterable(module.productions())) {
+            if (prod.att().contains("mlOp")) {
```

Comment:
I would rather we call the function that enumerates the klabels than check an attribute like this.